src/Mod/Sketcher/Gui/DrawSketchControllableHandler.h

```diff
@@ -102,6 +102,17 @@ class DrawSketchControllableHandler
         DSDefaultHandler::finish();
         return true;
     }
+
+    bool shouldDrawPositionAtCursor() const override
+    {
+        return toolWidgetManager.shouldDrawPositionAtCursor();

```

Comment:
These two functions, is what I was referring to when saying imposing interface requirements on the controller. As said, not inherently wrong. Just commenting implications.